backend/app/main.py

```diff
@@ -1084,6 +1086,32 @@ async def get_projects(
     return gerrydb_views
 
 
+@app.get("/api/document/{document_id}/thumbnail", status_code=status.HTTP_200_OK)
+async def get_thumbnail(
+    *,
+    document_id: str,
+    session: Session = Depends(get_session),
+):
+    try:
+        if thumbnail_exists(document_id):
+            return RedirectResponse(url=f"https://tilesets1.cdn.districtr.org/thumbnails/{document_id}.png")
```

Comment:
H: Please use an environment variable and settings `from app.core.config import settings` instead of hardcoding the base CDN url

Reply:
Do I need to add this to settings? I will use `{settings.CDN_URL}` for now